src/traci/PosixLauncher.cc

```diff
@@ -138,6 +141,13 @@ std::string PosixLauncher::command()
     command = std::regex_replace(command, port, std::to_string(m_port));
     command = std::regex_replace(command, seed, std::to_string(m_seed));
     command = std::regex_replace(command, run, run_number);
+
+    if (!m_options.empty()) {
+      std::ostringstream oss;
+      oss << command << " " << m_options;
+      command = oss.str();
```

Comment:
I prefer `command.append(1, ' ').append(m_options)` over streams here.

Reply:
Done; with only a couple of input strings streams are not necessary here.